core/api.txt

```diff
@@ -424,7 +424,13 @@ ion-datetime,event,ionChange,DatetimeChangeEventDetail,true
 ion-datetime,event,ionFocus,void,true
 ion-datetime,css-prop,--background
 ion-datetime,css-prop,--background-rgb
+ion-datetime,css-prop,--picker-fade-background-rgb
+ion-datetime,css-prop,--picker-highlight-background
 ion-datetime,css-prop,--title-color
+ion-datetime,part,picker-item
+ion-datetime,part,picker-item active
```

Comment:
Bikeshed: Can we use `wheel` terminology instead of `picker` to be consistent with the `preferWheel` property?

Reply:
Works for me 👍